devtools/gradle/gradle-model/src/main/java/io/quarkus/gradle/dependency/ApplicationDeploymentClasspathBuilder.java

```diff
@@ -288,6 +288,10 @@ public Configuration getRuntimeConfiguration() {
         return project.getConfigurations().getByName(this.runtimeConfigurationName);
     }
 
+    public Configuration getRuntimeConfigurationWithoutResolvingDeployment() {
+        return project.getConfigurations().getByName(this.runtimeConfigurationName);
```

Comment:
@cdsap do I understand correctly that the Quarkus runtime configuration will not include conditional dependencies with this change?

Reply:
It doesn't include conditional dependencies and somehow it appears to be not a regression.

Reply:
This method looks the same as the existing `getRawRuntimeConfiguration()` though